apps/web/src/views/BuyCrypto/hooks/useOnRampSignature.ts

```diff
@@ -68,7 +68,7 @@ export const useOnRampSignature = <selectData = GetOnRampSignatureReturnType>(
     ]),
     enabled: Boolean(externalTransactionId && quote && walletAddress && getIsNetworkEnabled(chainId)),
     queryFn: async () => {
-      if (!quote || !walletAddress || !externalTransactionId || !chainId || !onRampUnit) {
+      if (!quote || !walletAddress || !externalTransactionId || chainId === undefined || !onRampUnit) {
```

Comment:
We can add `chainId` in enabled Boolean in line `69`.
And remove the `undefined`

Reply:
@ChefMomota  no we cant sadly thats what the fix is. chainId is 0 for btc, Boolean(0) is false. so for chainiD I specifically need to check for undefined. or else this will throw when chainId is 0 for btc. thats why i changed this.